src/steadystate_adjoint.jl

```diff
@@ -126,42 +103,27 @@ end
     else
         if g !== nothing
             if dgdp_val !== nothing
-                gradient!(vec(dgdu_val),
-                          diffcache.g[1],
-                          y,
-                          sensealg,
+                gradient!(vec(dgdu_val), diffcache.g[1], y, sensealg,
                           diffcache.g_grad_config[1])
             else
-                gradient!(vec(dgdu_val),
-                          diffcache.g,
-                          y,
-                          sensealg,
-                          diffcache.g_grad_config)
+                gradient!(vec(dgdu_val), diffcache.g, y, sensealg, diffcache.g_grad_config)
             end
         end
     end
 
     if !needs_jac
-        # NOTE: Zygote doesn't support inplace
-        linear_problem = LinearProblem(VecJacOperator(f, y, p;
-                                                      autodiff = !DiffEqBase.isinplace(sol.prob)),
-                                       vec(dgdu_val),
-                                       u0 = vec(λ))
+        # TODO: FixedVecJacOperator should respect the `autojacvec` of the algorithm
+        operator = FixedVecJacOperator(f, y, p, Val(DiffEqBase.isinplace(sol.prob)))
+        linear_problem = LinearProblem(operator, vec(dgdu_val); u0 = vec(λ))
     else
-        linear_problem = LinearProblem(diffcache.J', vec(dgdu_val'), u0 = vec(λ))
+        linear_problem = LinearProblem(diffcache.J', vec(dgdu_val'); u0 = vec(λ))
     end
 
-    solve(linear_problem, linsolve) # u is vec(λ)
+    # Zygote pullback function won't work with deepcopy
+    solve(linear_problem, linsolve; alias_A = !needs_jac) # u is vec(λ)
```

Comment:
why not just true?

Reply:
I was not sure why the default is `false`. So wanted to keep it that for cases where the default works

Reply:
It's a safety measure if the `A` is reused, but it's not here.